docs/faq.md

```diff
@@ -8,7 +8,7 @@
   and less error-prone to write, for the entire stack, not
   just the end application code.
 
-  Typically many middleware would
+  Typically many middlewares would
```

Comment:
I disagree, the plural form of middleware is middleware.

Reply:
I updated the PR with the changes. I looked middleware and found a debatable answer. Both middleware and middlewares are used. But since middleware is uncountable as per english dictionary, let's use middleware for both singular and plural form moving forward. @niftylettuce what do you say?